scripts/customRulesTests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I kind of just took [`tslint-reacts`' approach](https://github.com/palantir/tslint-react/blob/master/scripts/verify.sh#L1)

@GoodForOneFare I remember you had different implementation in mind during our pairing session. What was it again? Just curious.